public/app/plugins/datasource/elasticsearch/datasource.ts

```diff
@@ -534,13 +536,15 @@ export class ElasticDatasource
     switch (type) {
       case SupplementaryQueryType.LogsVolume:
         return this.getLogsVolumeDataProvider(request);
+      case SupplementaryQueryType.LogsSample:
+        return this.getLogsSampleDataProvider(request);
       default:
         return undefined;
     }
   }
 
   getSupportedSupplementaryQueryTypes(): SupplementaryQueryType[] {
-    return [SupplementaryQueryType.LogsVolume];
+    return [SupplementaryQueryType.LogsVolume, SupplementaryQueryType.LogsSample];
   }
 
   getSupplementaryQuery(options: SupplementaryQueryOptions, query: ElasticsearchQuery): ElasticsearchQuery | undefined {
```

Comment:
Could you please update `elasticsearch/datasource.test.ts` covering the code added in this PR? Thanks! 

https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/elasticsearch/datasource.test.ts#L926